public/app/features/admin/ldap/LdapDrawer.tsx

```diff
@@ -181,67 +211,203 @@ export const LdapDrawerComponent = ({ onClose }: Props) => {
                 'If set to true, use LDAP with STARTTLS instead of LDAPS'
               )}
             >
-              <Switch id="start-tls" {...register('settings.config.servers.0.start_tls')} />
+              <Switch id="start-tls" {...register(`${serverConfig}.start_tls`)} />
             </Field>
             <Field
               htmlFor="min-tls-version"
-              label={t('ldap-drawer.extra-security-section.min-tls-version.label', 'Min TLS version')}
+              label={t('ldap-drawer.extra-security-section.min-tls-version-label', 'Min TLS version')}
               description={t(
-                'ldap-drawer.extra-security-section.min-tls-version.description',
+                'ldap-drawer.extra-security-section.min-tls-version-description',
                 'This is the minimum TLS version allowed. Accepted values are: TLS1.2, TLS1.3.'
               )}
             >
               <Select
                 id="min-tls-version"
                 options={tlsOptions}
-                value={watch('settings.config.servers.0.min_tls_version')}
-                onChange={({ value }) => setValue('settings.config.servers.0.min_tls_version', value)}
+                value={watch(`${serverConfig}.min_tls_version`)}
+                onChange={({ value }) => setValue(`${serverConfig}.min_tls_version`, value)}
               />
             </Field>
             <Field
-              label={t('ldap-drawer.extra-security-section.tls-ciphers.label', 'TLS ciphers')}
+              label={t('ldap-drawer.extra-security-section.tls-ciphers-label', 'TLS ciphers')}
               description={t(
-                'ldap-drawer.extra-security-section.tls-ciphers.description',
+                'ldap-drawer.extra-security-section.tls-ciphers-description',
                 'List of comma- or space-separated ciphers'
               )}
             >
               <Input
                 id="tls-ciphers"
                 placeholder={t(
-                  'ldap-drawer.extra-security-section.tls-ciphers.placeholder',
+                  'ldap-drawer.extra-security-section.tls-ciphers-placeholder',
                   'e.g. ["TLS_AES_256_GCM_SHA384"]'
                 )}
-                value={watch('settings.config.servers.0.tls_ciphers')}
+                value={watch(`${serverConfig}.tls_ciphers`) || ''}
                 onChange={({ currentTarget: { value } }) =>
                   setValue(
-                    'settings.config.servers.0.tls_ciphers',
-                    value?.split(/,|\s/).map((v: string) => v.trim())
+                    `${serverConfig}.tls_ciphers`,
+                    value?.split(/,|\s/).map((v) => v.trim())
                   )
                 }
               />
             </Field>
+            <Field
+              label={t(
+                'ldap-drawer.extra-security-section.encryption-provider-label',
+                'Encryption key and certificate provision specification (required)'
```

Comment:
I think you can remove the ` (required)` part from the title here. It's confusing because this field is always set (it's base64 or path), you cannot unset it.

Reply:
changed as requested